File: src/main/java/com/ocds/access/model/dto/tender/Item.java
package com.ocds.access.model.dto.tender;

import lombok.Data;

import java.util.List;

@Data
public class Item {
    public String id;
    public String description;
    public Classification classification;
    public List<Classification> additionalClassifications;
    public Integer quantity;
    public Unit unit;
}

File: src/main/kotlin/com/procurement/access/model/dto/ocds/Item.kt
package com.procurement.access.model.dto.ocds

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude
import java.math.BigDecimal

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Item @JsonCreator constructor(

    var id: String?,

    val internalId: String?,

    var description: String?,

    val classification: Classification,

    val additionalClassifications: List<Classification>?,

    val quantity: BigDecimal,

    val unit: Unit,

    var relatedLot: String
)